zqd/api/api.go

```diff
@@ -98,7 +99,7 @@ func (s *SpaceID) Set(str string) error {
 type SpaceInfo struct {
 	ID          SpaceID      `json:"id"`
 	Name        string       `json:"name"`
-	DataPath    string       `json:"data_path"`
+	DataPath    iosrc.URI    `json:"data_path"`
```

Comment:
I think this should also change to data_uri but didn't want to force an api migration.